build-configs.yaml

```diff
@@ -248,6 +248,9 @@ build_configs_defaults:
         extra_configs: ['allmodconfig', 'allnoconfig']
         fragments: [x86_kvm_guest]
 
+reference_defaults:
```

Comment:
This should be made part of `build_configs_defaults` above, as it's passed as a special set of defaults when creating build configurations.  Then the code could look for the `reference` entry there if it's not in a given build configuration.